.travis.yml

```diff
@@ -9,7 +9,17 @@ android:
       - platform-tools
       - build-tools-25.0.0
       - android-25
+      - android-22
```

Comment:
Can you add 

> Android 22 is the latest in Travis at this time: https://docs.travis-ci.com/user/languages/android/

comment around here so that it will serve as a reminder for the next person looking at here?

Also, would not this make the one previous line obsolete? Having to similar lines one after another looks suspicious to me (I am not familiar with Travis build).